build/Packages.props

```diff
@@ -33,20 +35,20 @@
         <PackageReference Update="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="$(RoslynPackageVersion)" />
         <PackageReference Update="Microsoft.CodeAnalysis.Workspaces.Common" Version="$(RoslynPackageVersion)" />
 
-        <PackageReference Update="Microsoft.Extensions.Caching.Memory" Version="3.1.12" />
-        <PackageReference Update="Microsoft.Extensions.Configuration" Version="3.1.12" />
-        <PackageReference Update="Microsoft.Extensions.Configuration.Binder" Version="3.1.12" />
-        <PackageReference Update="Microsoft.Extensions.Configuration.CommandLine" Version="3.1.12" />
-        <PackageReference Update="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="3.1.12" />
-        <PackageReference Update="Microsoft.Extensions.Configuration.Json" Version="3.1.12" />
-        <PackageReference Update="Microsoft.Extensions.DependencyInjection" Version="3.1.12" />
+        <PackageReference Update="Microsoft.Extensions.Caching.Memory" Version="$(MicrosoftExtensionPackageVersion)" />
```

Comment:
We'll have to keep on things, sometimes these packages are versioned differently depending if a vulnerability or something is discovered.

Reply:
I can revert if you prefer. I just saw a wall of references with the same version number and my inclination was to consolidate. I didn't change the version being used by these packages.

Reply:
Nah it's fine, just something we need to keep in mind in the future.  I don't think we're going to be running the latest versions all the time.